README.md

```diff
@@ -1,10 +1,24 @@
 # Lora Based BraveSensor Firmware
 
+# Project Overview
+
+## Setup
+Follow this [guide](https://docs.rakwireless.com/Product-Categories/WisDuo/RAK3272S-Breakout-Board/Quickstart/#rak3272s-breakout-board-as-a-stand-alone-device-using-rui3) on how to setup. The LoraBraveSensor directory can subsitute the Arduino Serial demo, as a guide on how to compile and upload our custom firmware. After going throught the entire process, I recommend using vscode's Arduino extension for easier file management. 
+
+## Secrets File
+DEVEUI: which can be found by scanning the QR code on the LoRaWan chip. 
+APPEUI: currently we use the same APPEUI as the one for BraveButtons
+APPEUI: APPEUI = hex(str(hex(DEVEUI)) + str(hex(APPEUI)))
+An example one can be found in our private [living doc](https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-8442)
+
+### Using Arduino Extension for VSCode
+I recommend using VSCode's Arduino extension for easier file management, after going through the setup process with the Arduino IDE at least once. After opening the project with VSCode, if you notice red squiggly error messages for missing includes, follow [this](https://stackoverflow.com/questions/52234438/vs-code-giving-header-errors-for-arduino-missing-official-header) guide on how to fix them. On Windows 10, the board firmware files can be found at `C:/Users/{user}/AppData/Local/Arduino15/packages/rak_rui/hardware/stm32/3.4.2/cores/STM32WLE/component/rui_v3_api`, and can be added under `includePath` of the `.vscode/c_cpp_properties.json` file. 
+
+## Serial Debugging
+Can be done with any serial monitor (e.g. Arduino, Arduino's VSCode Extension, PlatformIO, etc.) with BAUD rate of 115200. 
```

Comment:
Are there values that need to change in the code to turn this on or off?

Reply:
There is, I'll add it here.